src/app/view/components/displayTimerText.js

```diff
@@ -0,0 +1,22 @@
+import elementCreator from '../../utils/elementCreator';
+
+// const tylko do testów, docelowo time powinien być pobierany z zewnątrz
+const time = 45;
+
+export default function displayTimerText() {
+  const min = Math.floor(time / 60);
+  const sec = time % 60;
+
+  const timer = elementCreator('p', { class: 'game-timer__text' });
+
+  if (time <= 20) {
+    timer.style.color = 'rgba(255, 0, 0, 0.8)';
+  }
+
+  if (time < 10 || time % 60 === 0) {
+    timer.innerHTML = `Time Left: ${min}m 0${sec}s`;
```

Comment:
I'm not sure but if You don't pack any HTML tag on the right side you can use innerText. 
if we left many innerHTML places some ninja hackers can inject here some dom elements.
For example:

some ninja hacked out timer and inject this in innerHTML:
`<img src="http://photos.com/golababka.jpg">`
and in our timer place, we will see golababka.jpg image

if some ninja hacked out timer and inject this in innerText:
`<img src="http://photos.com/golababka.jpg">`
in our timer place we will see text: `"<img src="http://photos.com/golababka.jpg">"`

it's called XSS attack